generator/testdata/google/example/library/v1/library_service_stub.gapic.cc.baseline

```diff
@@ -124,14 +126,208 @@ class DefaultLibraryServiceStub : public LibraryServiceStub {
+      }
+
+      auto delay = backoff_policy->OnCompletion();
+      std::this_thread::sleep_for(delay);
+    }
+  }
+
+  google::gax::Status
+  DiscussBook(google::gax::CallContext& context,
+             ::google::example::library::v1::DiscussBookRequest const& request,
+             ::google::example::library::v1::Comment& response) {
+    auto retry_policy = (ctx.retry_policy_ ? ctx.retry_policy_ : retry_policy_)->clone();
+    auto backoff_policy = (ctx.backoff_policy_ ? ctx.backoff_policy_ : backoff_policy_)->clone();
+    gax::MethodInfo info = ctx.Info();
+    while (true) {
+      retry_policy->Setup(context);
+      gax::Status status = next_stub_->DiscussBook(context, request, response);
+      if (status.IsOk() || !retry_policy->OnFailure(status) ||
+          !info.idempotency == gax::MethodInfo::Idempotency::IDEMPOTENT) {
+        return status;
+      }
+
+      auto delay = backoff_policy->OnCompletion();
+      std::this_thread::sleep_for(delay);
+    }
+  }
+
+  google::gax::Status
+  MonologAboutBook(google::gax::CallContext& context,
+             ::google::example::library::v1::DiscussBookRequest const& request,
+             ::google::example::library::v1::Comment& response) {
+    auto retry_policy = (ctx.retry_policy_ ? ctx.retry_policy_ : retry_policy_)->clone();
+    auto backoff_policy = (ctx.backoff_policy_ ? ctx.backoff_policy_ : backoff_policy_)->clone();
+    gax::MethodInfo info = ctx.Info();
+    while (true) {
+      retry_policy->Setup(context);
+      gax::Status status = next_stub_->MonologAboutBook(context, request, response);
+      if (status.IsOk() || !retry_policy->OnFailure(status) ||
+          !info.idempotency == gax::MethodInfo::Idempotency::IDEMPOTENT) {
+        return status;
+      }
+
+      auto delay = backoff_policy->OnCompletion();
+      std::this_thread::sleep_for(delay);
+    }
+  }
+
+  google::gax::Status
+  GetBigBook(google::gax::CallContext& context,
+             ::google::example::library::v1::GetBookRequest const& request,
+             ::google::example::library::v1::Book& response) {
+    auto retry_policy = (ctx.retry_policy_ ? ctx.retry_policy_ : retry_policy_)->clone();
+    auto backoff_policy = (ctx.backoff_policy_ ? ctx.backoff_policy_ : backoff_policy_)->clone();
+    gax::MethodInfo info = ctx.Info();
+    while (true) {
+      retry_policy->Setup(context);
+      gax::Status status = next_stub_->GetBigBook(context, request, response);
+      if (status.IsOk() || !retry_policy->OnFailure(status) ||
+          !info.idempotency == gax::MethodInfo::Idempotency::IDEMPOTENT) {
+        return status;
+      }
+
+      auto delay = backoff_policy->OnCompletion();
+      std::this_thread::sleep_for(delay);
+    }
+  }
+
+ private:
+  std::unique_ptr<LibraryServiceStub> next_stub_;
+  const std::unique_ptr<gax::RetryPolicy const> retry_policy_;
+  const std::unique_ptr<gax::BackoffPolicy const>  backoff_policy_;
+}; // RetryLibraryServiceStub
 } // namespace
```

Comment:
nit: the style guide asks for two spaces before the namespace comment.  Consider setting up clang-format(1) for this repo, you can copy our setup if that helps:

https://github.com/googleapis/google-cloud-cpp/blob/master/.clang-format

Reply:
Will fix the namespace comment issue.

As a heads up, formatting for the generated surface is very much a best effort attempt. Correct spacing based on identifier length is a lot more sophisticated than I think we can reasonably handle using the proto formatting library.

Reply:
Ack.